background/messages/mCreateWindow.ts

```diff
@@ -3,8 +3,9 @@ import { runtime, tabs, windows } from "webextension-polyfill"
 import { type PlasmoMessaging } from "@plasmohq/messaging"
 
 export type RequestBody = {
-  origin: string
-  account: string
+  user?: string
+  chalBase64Url?: string
```

Comment:
What is `chal` for? I think we can use full name in the variables for readability.